krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
I'm not sure that making a syscall so frequently is a great idea (time.clock(), for example, will interfere with `ntp`). Maybe stick a dummy for loop in there as well? Even if we just make the syscall 100 times a second, that'll probably be better.